### ISSUE TITLE
Incompatible with 2022.1 EAP

### ISSUE THREAD

**Issue by EtienneBruines** [JetBrains/idea-gitignore#747]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [x] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description

The plugin is not (marked as) compatible with 2022.1 EAP, despite subscribing to the EAP-channel of the plugin (which hasn't been updated in a while). 

(It is always *this* plugin that has issues with marked compatibility for new EAP releases - is there a way that the EAP version of the plugin can be marked as "compatible with all" or something like that? To not require changes after every new release?)

**Comment by hsz** [comment]
On it! I'll release a compatible version soon.

**Comment by DAmNRelentless** [comment]
"soon"?

**Comment by michal-kocarek** [comment]
It would be great if JetBrains allocated a little bit time for this plugin to give it some love and enable it for 2022 version. It is right now the only one blocking me from using the EAP...

**Comment by mojotx** [comment]
I guess I am uninstalling GoLand 2022.1 EAP, and rolling back to the stable version, since a compatible version of this plugin still hasn't been released.

**Comment by mojotx** [comment]
I checked the logfile, and see this:

```
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.3427.90/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.3427.90
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.3427.90/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.3427.90
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.3427.90/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.3427.90
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.3427.90/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.3427.90
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.4994.43/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.4994.43
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.4994.43/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.4994.43
idea.log:  The .ignore (id=mobi.hsz.idea.gitignore, path=~/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/221.4994.43/GoLand 2022.1 EAP.app.plugins/.ignore, version=4.3.0) plugin Plugin '.ignore' (version '4.3.0') is not compatible with the current version of the IDE, because it requires build 213.* or older but the current build is GO-221.4994.43
```

**Comment by mojotx** [comment]
Still incompatible with GoLand 2022.1 Beta.

**Comment by hsz** [comment]
Fixed with v4.4.0
